buildbot/configure.py

```diff
@@ -327,6 +330,9 @@ def main():
         default="AMD",
         help="choose hardware platform for HIP backend",
     )
+    parser.add_argument(
+        "--level_zero_v2", action="store_true", help="Enable SYCL level_zero_v2"
```

Comment:
Slight preference to not have variable names in user help
```suggestion
        "--level_zero_v2", action="store_true", help="Enable SYCL Level Zero V2"
```

Reply:
Good point, changed that